Automattic/wp-calypso
# Plan Features FAQ: Include eCommerce Plan

#### Changes proposed in this Pull Request

* Includes mention of the eCommerce plan for the Plan Features FAQ

#### Testing instructions

On a WordPress.com site, visit the Plans page and scroll to the bottom, then verify the copy changes.

<img width="1120" alt="Screenshot 2019-07-17 at 17 07 28" src="https://user-images.githubusercontent.com/43215253/61391887-9abaef80-a8b5-11e9-93fd-a695d8ce37dd.png">

Fixes #34721
